src/cmd/ksh93/meson.build

```diff
@@ -22,7 +27,6 @@ ksh93_c_args = ['-DERROR_CATALOG=0',
                 '-DSHOPT_MULTIBYTE',
                 '-DSHOPT_NAMESPACE',
                 '-DSHOPT_OPTIMIZE',
-                '-DSHOPT_PFSH',
```

Comment:
I agree with removing this. Enabling features for a specific OS (as opposed to working around bugs or limitations of a specific OS) does not belong in the generic project.